lean/models/brokerages/cloud/binance.py

```diff
@@ -43,15 +44,18 @@ def build(cls, logger: Logger) -> CloudBrokerage:
 Demo credentials can be generated on Binance Testnet (https://testnet.binance.vision/).
         """.strip())
 
+        exchange_name = click.prompt("Binance Exchange [Binance|BinanceUS]")
```

Comment:
Shouldn't these be choices versos having the user to type it out? Same for FTX really